clusters/base/kepler.yaml

```diff
@@ -20,15 +20,18 @@ metadata:
 spec:
   interval: 24h
   # Values.yaml updated. Refer: https://fluxcd.io/flux/components/helm/helmreleases/#inline-values
+
   values:
     serviceMonitor:
       enabled: "true"
       namespace: "monitoring"
+    nodeSelector:
+      kubernetes.io/os: linux
```

Comment:
Why is this needed?

Reply:
It was in kepler docs that's why
I think we can remove it

Reply:
Thank you for clarifying. We only plan to support linux nodes so yes let's remove it.